### DISCUSSION TITLE
Prometheus (and possibly Munin) Exporter for Monika

### DISCUSSION THREAD

**Discussion by newm4n** [hyperjumptech/monika#138] [Ideas]
Received feedback from reddit DevOps 
"Would be nice as prometheus exporter".

Short investigation to https://prometheus.io/docs/instrumenting/exporters/#other-monitoring-systems
shows lots of monitoring tools also exports metrics to prometheus.
This is useful to see Monika's metrics in prometheus graph

As how to write an exporter, 

- Look for those opensource projects that did prometheus exporter
- https://prometheus.io/docs/instrumenting/writing_exporters/

Another possible NICE TO HAVE is an exporter for MUNIN (http://munin-monitoring.org/). 
Its a simpler stright-forward kind of prometheus.

All of those, must be done in JavaScript for simplicity in context of Monika.

**Comment by sapiderman** [comment]
As we're using pino.js for logging, it already supports several tranports

https://github.com/pinojs/pino/blob/master/docs/transports.md

No prometheus yet though, but some of these others might be interesting to potential users.
